pkgs/tools/misc/zellij/default.nix

```diff
@@ -30,11 +31,19 @@ rustPlatform.buildRustPackage rec {
     mandown
     installShellFiles
     pkg-config
+    (lib.getDev curl)
```

Comment:
For curl-config, right?

Reply:
So that cargo finds curl, using pkg-config. Without this, it doesn't.